### ISSUE TITLE
Facing error:

### ISSUE THREAD

**Issue by Jashwanth-Raju-03** [Pika1998/face-recognition-login-system#7]
**Hi bro I'm used you're face attendance project, it was cool but I'm facing issue in that the code is successfully running but while pressing the login but it will take me to log-in.py code.... Can you solve it please?**
![Screenshot from 2023-01-19 21-10-00](https://user-images.githubusercontent.com/122804504/214886705-9a0e42b7-b2e3-4c9a-82fe-40a1f858cd4e.png)

**Comment by prafulla-codes** [comment]
Hey, Let me check on the same

**Comment by Jashwanth7904** [comment]
can you solve it bro..? please...!!,its urgent for my clg project bro

**Comment by prafulla-codes** [comment]
Hey @Jashwanth7904 , have you installed CGI?

**Comment by Jashwanth-Raju-03** [comment]
No, I haven't. Bro okay okay that's the mistake bro..?  For this error

**Comment by Jashwanth-Raju-03** [comment]
Where should I have to install cgi bro, I mean in wich path..?

**Comment by Jashwanth-Raju-03** [comment]
Bro, are you CGI.py file..? or I have to install it separately..?

**Comment by prafulla-codes** [comment]
Hey, can you go through the docs again and ensure that you have done all the steps 
And modified the files required for XAMPP

**Comment by Jashwanth-Raju-03** [comment]
Sure, will do that.

**Comment by prafulla-codes** [comment]
Hey @Jashwanth7904 , is this resolved?

**Comment by Jashwanth-Raju-03** [comment]
No, it doesn't.
